runtime/lua/vim/filetype.lua

```diff
@@ -949,6 +949,55 @@ local extension = {
     end
     vim.fn["dist#ft#SetFileTypeSH"]("bash")
   end,
+  pm = function(path, bufnr)
+    if getline(bufnr, 1):find("XPM2") then
+      return "xpm2"
+    elseif getline(bufnr, 1):find("XPM") then
+      return "xpm"
+    else
+      return "perl"
+    end
+  end,
+  me = function()
+    if fn.expand('<afile>') ~= 'read.me' and fn.expand('<afile>') ~= 'click.me' then
```

Comment:
Use `path` (the first argument passed to this function) instead of `<afile>`.

Reply:
In this case to get the filename I would do `vim.fn.fnamemodify(path, ":t")`, right?

Reply:
Yep!